.github/workflows/ci.yaml

```diff
@@ -64,6 +63,31 @@ jobs:
         shell: bash
         run: .github/workflows/test.sh ${{ runner.os }}
 
+      - if: runner.os == 'Linux'
```

Comment:
Is it worth splitting this out into a separate job so that in the CI UI it says ubuntu, macOS, windows, android?

Reply:
Yeah I wasn't sure about this. The build:

- Breaks on Mac because of the wrong NDK name;
- Breaks on Windows because `export` is not a thing there.

The former one should be fixed now, the latter one I don't immediately know how to resolve (without writing separate scripts for separate os'es). Does Windows understand `$HOME` as used elsewhere in this file, shouldn't that be `%UserProfile%`?

The main reason it' stuffed in `check` is to leverage existing setup and build artifacts: is that okay or is there a better solution in the pipeline with the changes discussed yesterday?

Reply:
Should be fixed now: we don't need any of these ugly download steps as the GH Actions image comes with the NDK preinstalled :tada: 

Still an issue with spaces in paths on Windows, which I hope to get to the bottom of here: https://github.com/rust-windowing/android-ndk-rs/pull/92

Reply:
Spacing issue should be solved now, cc https://github.com/rust-windowing/android-ndk-rs/pull/92#issuecomment-723455010.